sonic_platform_base/psu_base.py

```diff
@@ -27,7 +27,7 @@ class PsuBase(device_base.DeviceBase):
     _thermal_list = []
 
     # Status of Master LED
-    psu_master_led_color = STATUS_LED_COLOR_OFF
+    psu_master_led_color = device_base.DeviceBase.STATUS_LED_COLOR_OFF
```

Comment:
Why is this necessary? Can we not use `self.` instead, since this constant should be inherited?

Reply:
Here is the class level variable and cannot use the key word ```self```  here.

Reply:
Have you tested this? I believe it should be possible via inheritance. See https://stackoverflow.com/a/10672749

Reply:
Yes, I tested it. A NameError will be reported. I did a small test just now:

```python
class A:
    VAR = 1


class B(A):
    VAR1 = VAR

b = B()
```

And the output:

```
Traceback (most recent call last):
  File "D:/code/test/test1.py", line 5, in <module>
    class B(A):
  File "D:/code/test/test1.py", line 6, in B
    VAR1 = VAR
NameError: name 'VAR' is not defined
```

Reply:
You didn't add `self.`. Try this:

```python
class A:
    VAR = 1


class B(A):
    VAR1 = self.VAR

b = B()
```

Reply:
Got the same:

```python
class A:
    VAR = 1


class B(A):
    VAR1 = self.VAR

b = B()
```

Output:
```
Traceback (most recent call last):
  File "D:/code/test/test1.py", line 5, in <module>
    class B(A):
  File "D:/code/test/test1.py", line 6, in B
    VAR1 = self.VAR
NameError: name 'self' is not defined
```

Reply:
Ah, because `self` is only available in a function. Can you set `psu_master_led_color = None` here and add `self.psu_master_led_color = self.STATUS_LED_COLOR_OFF` in the constructor?

Reply:
sure

Reply:
Actually, you can probably omit the `psu_master_led_color = None` line.

Reply:
Fixed